.github/workflows/netlify.yml

```diff
@@ -0,0 +1,83 @@
+name: "📤 Upload Preview Build to Netlify"
+on:
+  workflow_run:
+    workflows: ["🌐 Build the matrix.org website"]
+    types:
+      - completed
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: >
+      ${{ github.event.workflow_run.conclusion == 'success' }}
+    steps:
+      # There's a 'download artifact' action but it hasn't been updated for the
+      # workflow_run action (https://github.com/actions/download-artifact/issues/60)
+      # so instead we get this mess:
+      - name: "📥 Download artifact"
+        uses: actions/github-script@v3.1.0
+        with:
+          script: |
+            var artifacts = await github.actions.listWorkflowRunArtifacts({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              run_id: ${{github.event.workflow_run.id }},
+              });
+              var matchArtifact = artifacts.data.artifacts.filter((artifact) => {
```

Comment:
there's something weird going on with the indentation here.

Reply:
Should be fixed.